package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "frontend",
-  "version": "1.6.1",
+  "version": "1.6.2",
```

Comment:
@Pancakem  you can take this to version 1.7.0 this is big enough  of a feature

Reply:
@zacck done.